megamek/src/megamek/client/ui/swing/CommonSettingsDialog.java

```diff
@@ -156,32 +165,32 @@ private <T> void moveElement(DefaultListModel<T> srcModel, int srcIndex, int trg
     }
 
     private final JCheckBox autoEndFiring = new JCheckBox(Messages.getString("CommonSettingsDialog.autoEndFiring"));
-    private final JCheckBox autoDeclareSearchlight = new JCheckBox(
-            Messages.getString("CommonSettingsDialog.autoDeclareSearchlight"));
+    private final JCheckBox autoDeclareSearchlight = new JCheckBox(Messages.getString(
+          "CommonSettingsDialog.autoDeclareSearchlight"));
```

Comment:
Is this the standard going forward?  I like how compact this is, but the double parens on the same line seem like they'll get really long in a few places.

Reply:
This is the EC changes due to line length. It can be worked on still or refactored in some way with the I18N changes.